packages/cli/src/lib/decorators/preconditions/apiKeyPrivilege.ts

```diff
@@ -0,0 +1,93 @@
+import PlatformClient, {
+  PrivilegeEvaluatorModel,
+} from '@coveord/platform-client';
+import Command from '@oclif/command';
+import dedent from 'ts-dedent';
+import {Config} from '../../config/config';
+import {AuthenticatedClient} from '../../platform/authenticatedClient';
+
+export const impersonatePrivilege = {
+  requestedPrivilege: {
+    owner: 'SEARCH_API',
+    targetDomain: 'IMPERSONATE',
+    targetId: '*',
+  },
+};
+
+export const createApiKeyPrivilege = {
+  requestedPrivilege: {
+    owner: 'PLATFORM',
+    targetDomain: 'API_KEY',
+    targetId: '*',
+    type: 'CREATE',
+  },
+};
+
+export function HasNecessaryCoveoPrivileges() {
+  return async function (target: Command) {
+    const authenticatedClient = new AuthenticatedClient();
+
+    const client = await authenticatedClient.getClient();
+    const {organization, anonymous} = await getConfiguration(target);
+
+    console.log('*********************');
```

Comment:
Why those logs here ?

Reply:
oups! I was doing some testing and forgot to remove it.